import_data/tasks/tasks.py

```diff
@@ -852,7 +852,7 @@ def run_osm_update(ctx):
 
         new_osm_timestamp = read_osm_timestamp(ctx, change_file_path)
         ctx.run(
-            f"{os.path.join(os.getcwd(), 'osm_update.sh')} --config {ctx.imposm_config_dir} --input {change_file_path}",
+            f"python3 {os.path.join(os.getcwd(), 'osm_update.py')} --config {ctx.imposm_config_dir} --input {change_file_path}",
```

Comment:
That's not a strong requirement, but we could probably avoid calling a separate script here, now that the main logic is implemented in python.  
For example it would be possible to reuse the current settings `ctx`, and we would no longer need any args parsing.

Reply:
We have the choice, I first decided to do it like this to keep the same behavior, but it seems better to just import the function and call it. So I updated the code this way, no need to create a new process anymore.